open_samus_returns_rando/files/randomizer_powerup.lua

```diff
@@ -139,192 +126,161 @@ function RandomizerPowerup.IncreaseAmmo(resource)
     RandomizerPowerup.IncreaseItemAmount(current_id, resource.quantity, resource.item_id)
 end
 
--- Main PBs (always) + PB expansions (if required mains are disabled)
-function RandomizerPowerup._AddLockedPBS(main_item)
-    local locked_pbs = RandomizerPowerup.GetItemAmount("ITEM_RANDO_LOCKED_PBS")
-    local current_max = RandomizerPowerup.GetItemAmount("ITEM_WEAPON_POWER_BOMB_MAX")
-    local current_current = RandomizerPowerup.GetItemAmount("ITEM_WEAPON_POWER_BOMB_CURRENT")
-    local new_current = 0
-    if main_item then
-        new_current = current_max + locked_pbs
-    else
-        new_current = current_current + locked_pbs
-    end
-    RandomizerPowerup.SetItemAmount("ITEM_WEAPON_POWER_BOMB_MAX", current_max + locked_pbs)
-    RandomizerPowerup.SetItemAmount("ITEM_WEAPON_POWER_BOMB_CURRENT", new_current)
-    RandomizerPowerup.SetItemAmount("ITEM_RANDO_LOCKED_PBS", 0)
-    hud.UpdatePlayerInfo(true)
+-- TODO: What's the max energy?
```

Comment:
?

Reply:
1099

Reply:
Does the game do not explode at refill stations? Iirc that was the issue with dread: You could go higher but have crashes at refill stations.

Reply:
I don't recall having issues